### PR TITLE
Use 'centos:8' image as the base image of tidb-backup-manager

### PR DESCRIPTION
**Pull request by KanShiori**
### What problem does this PR solve?

Use 'centos:8' image as the base image of tidb-backup-manager.
### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [x] Manual test 
Use new *Dockerfile* to build images, and try to deploy tidb-backup-manager.
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: images/tidb-backup-manager/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM pingcap/tidb-enterprise-tools:latest
+FROM centos:8
 ARG RCLONE_VERSION=v1.51.0
 ARG SHUSH_VERSION=v1.4.0
 ARG TOOLKIT_VERSION=v5.0.1
-RUN apk update && apk add ca-certificates
+RUN yum makecache && yum install ca-certificates wget unzip -y
```
[the remark this answers is not in the document]
Comment (DanielZhangQD) [reply images/tidb-backup-manager/Dockerfile:5]: No need to make cache in the container, right?

[the remark this answers is not in the document]
Comment (KanShiori) [reply images/tidb-backup-manager/Dockerfile]: right, I have modified.

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**